scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -195,37 +208,29 @@ class ThemeAggregator {
     const { theme, scoped } = options;
 
     if (!theme && !scoped) {
-      Logger.warn('No theme provided.', { context });
+      Logger.warn('No theme provided.', { LOG_CONTEXT });
       return false;
     }
 
     return true;
   }
 
   /**
-   * Generates a theme scss file and outputs it to the generated-themes directory.
-   * @param {object} contains requested scss file attrs
-   *   @param {array} assets - The aggregated theme files to import within generated file.
-   *   @param {string} themeName - Name of theme to aggregate.
-   *   @param {string} prefix - Prefix to append to generated file.
-   *   @param {string} scopeSelector - scss scope selector to encase theme.
-   *   @param {string} outputPath - path to write the scss file to. For testing purposes - overrides the default generated-themes path.
-   * @returns {string} - The path of the generated scss file, relative to the working home directory.
+   * Searches for theme files located within the following directory format: themes->theme-name->theme-name.scss.
+   * @param {string} themeName - The theme to aggregate.
+   * @param {Object} options - The aggregation options.
+   * @returns {array} - An array of ${themeName} files.
    */
-  static writeSCSSFile({
```

Comment:
Why remove this function? It looks like it's been moved to `generateTheme`?

Reply:
To improve `generateTheme`'s cohesion. Calling `writeSCSSFile` seems to make `generateTheme` a shell of a function. I can see the other side of the argument - keeping `writeSCSSFile` would be better for maintainability and extensibility.

If we kept the same structure, `generateTheme` would call `findThemeVariableFilesForGeneration`, pack the parameters into an object, pass this object into `writeSCSSFile`,  then return the value returned from `writeSCSSFile`. `writeSCSSFile` seems a little too coupled with `generatedThemes`.

What are your thoughts?
```jsx
// Alternative
static generateTheme(themeName, themeScope, options) {
const fileAttrs = {
      assets: ThemeAggregator.findThemeVariableFiles(themeName, options),
      themeName,
      prefix: isScoped && !isRoot ? SCOPED : ROOT,
      scopeSelector: isScoped && !isRoot ? `.${themeName}` : `:${ROOT}`,
    };

if (fileAttrs.assets) {
  return ThemeAggregator.writeSCSSFile(fileAttrs);
}

return null;
```